tide-whisperer.go

```diff
@@ -78,8 +80,9 @@ var (
 
 const (
 	dataAPIPrefix             = "api/data "
-	medtronicLoopBoundaryDate = "2017-09-01"
+	medtronicLoopBoundaryDateUnix = 1504238400
```

Comment:
It doesn't make much sense to have a timestamp here, that's later converted to `time.Time`. Having a string is better for readability.